docs/ZSL.md

```diff
@@ -1,97 +0,0 @@
-# ZSL Proof of Concept
```

Comment:
Any reason why we are deleting this completely? we still have some queries on earliest version of ZSL and some users are keen